tests/integration/chain_setup.go

```diff
@@ -83,16 +105,19 @@ func CreateTx(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, user
 }
 
 type MessagesForUser struct {
-	User cosmos.User
-	Msgs []sdk.Msg
+	User               cosmos.User
```

Comment:
can we just call this struct `UserTx` its a bit confusing rn

Reply:
👍